iron-a11y-keys-behavior.html

```diff
@@ -411,14 +374,21 @@
           event.stopPropagation();
         }
 
-        keyBindings.forEach(function(keyBinding) {
-          var keyCombo = keyBinding[0];
-          var handlerName = keyBinding[1];
+        // if event has been already prevented, don't do anything
+        if (event.defaultPrevented) {
+          return;
+        }
 
-          if (!event.defaultPrevented && keyComboMatchesEvent(keyCombo, event)) {
+        var eventKey = normalizedKeyForEvent(event);
+        for (var i = 0; i < keyBindings.length; i++) {
+          var keyCombo = keyBindings[i][0];
+          var handlerName = keyBindings[i][1];
+          if (keyComboMatchesEvent(keyCombo, event, eventKey)) {
             this._triggerKeyHandler(keyCombo, handlerName, event);
+            // found the matching keycombo, we can exit the loop
+            return;
```

Comment:
Two conditions come to mind that need to be handled.

Condition one:

I have a behavior Foo that implements some `keyBindings`, and an element that applies the behavior but also binds to the same key:

```js
Polymer.FooBehavior = {
  keyBindings: {
    'a': '_onAPressedFoo'
  }
};

Polymer({
  is: 'x-uses-foo',

  behaviors: [Polymer.FooBehavior],

  keyBindings: {
    'a': '_onAPressed'
  }
});
```

In this condition, we want the handler to be called for the key binding registered by the behavior and the key binding registered by the element using the behavior.

Condition two:

Two of my keyBindings are different but overlap in such a way that they will both be called for the same key combination:

```js
Polymer({
  is: 'x-redundant-key-combos',

  keyBindings: {
    'a': '_onAPressed',
    'shift+a': '_onAPressed'
  }
});
```

In this condition, I believe that `shift+a` will also match pressing `a` (remember that this change short-circuits checking for modifier keys if the key combo does not express a modifier key). This outcome is potentially undesirable, but desirable or not, the correct handler to call first is ambiguous so both should probably be called.

WDYT?